src/Resources/doc/usage.md

```diff
@@ -220,3 +220,97 @@ Example:
     $response = $mailjet->post(Resources::$Email, ['body' => $body]);
 
 ```
+## Campaigndraft Example
+
+You can also access the [/campaigndraft](https://dev.mailjet.com/email-api/v3/campaigndraft) api through the CampaignDraftManager 
+
+Example:
+
+``` php
+<?php
+use \Mailjet\Resources;
+use Mailjet\MailjetBundle\Model\CampaignDraft;
+use Mailjet\MailjetBundle\Manager\CampaignDraftManager;
+// ...
+public function campaignDraftExample() {
+    // ...
+    $mailjet = $this->container->get('mailjet.client');
+    $campaignDraftManager = new CampaignDraftManager($mailjet);
+
+    $optionalProp['Title'] = 'Friday newsletter';
+    $optionalProp['SenderName'] = 'Mailjet team';
+    $optionalProp['EditMode'] = 'html2';
+    $campaignDraft = new CampaignDraft("en_US", "Lyubo", "latanasov@mailjet.com", "Symfony bundle test", "5410");
+    $campaignDraft->setOptionalProperties($optionalProp);
+    $ID = $campaignDraftManager->create($campaignDraft)[0]['ID'];
+
+    /*     * Get the ID from the newly created CampaignDraft* */
+    $campaignDraft->setId($ID);
```

Comment:
I'm basically creating a campaign draft, and retrieving the ID from the response in a single line.  Do you think it would be clearer if done in multiple statements ?